src/Areas/Server/Commands/ServiceStartCommand.cs

```diff
@@ -106,46 +96,17 @@ public override async Task<CommandResponse> ExecuteAsync(CommandContext context,
     /// <returns>An IHost instance configured for the MCP server.</returns>
     private IHost CreateHost(ServiceStartOptions serverOptions)
```

Comment:
Make sure you aren't being too aggressive in all that you have removed from CreateHost.  If you have a problem with any of that then add it back and fix it.